### PR TITLE
Create WIP_enviromental_inhalers.json

### PR DESCRIPTION
**Pull request by brianmackenna**
This is a work in progress. Descriptions and "why it matters" needs expanding.

This is a prototype for staging as I have never created measures using a view, I'm not certain it is fully correct.

### CODE REVIEW

File: openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "tags": [
+    "respiratory"
+  ],
+  "numerator_columns": [
+    "SUM(items) AS numerator, "
+  ],
+  "numerator_from": "{hscic}.brian.enviromental_inhalers",
+  "numerator_where": [
+   "(form_route='pressurizedinhalation.inhalation')",
+    "1 = 1"
```
Comment (sebbacon) [inline openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers]: You wouldn't need the 1 = 1 here.

**Comment by brianmackenna** [comment]
Is there a particular reason why `vw__measures` includes 2 underscores?

File: openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Enviromental Impact of Inhalers",
```
Comment (inglesp) [inline openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers]: Enviro_n_mental

**Comment by inglesp** [comment]
The file needs to be called `something.json`.  (Before it's deployed, you'll want to rename it anyway, as the filename becomes part of the URL of the measure.)

File: openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers (hunk 2 above)
Comment (brianmackenna) [reply openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers]: thanks and updated. I wonder how many years I have been spelling that wrong for.....

**Comment by inglesp** [comment]
@brianmackenna I'm pretty sure we don't need to use a view at all here -- are you happy for me to rewrite the measure definition?

**Comment by brianmackenna** [comment]
Please do - Once done if we could briefly chat on a call as well? Obviously I'm a bit unclear about views and the measure generation more generally.

**Comment by brianmackenna** [comment]
@inglesp  has rewritten a new PR #2038  with a new view so I will close this one.

**Pull request closed without merging**